src/defaultSettings.js

```diff
@@ -0,0 +1,132 @@
+const defaultSettings = {
+  //Custom settings for this template
+    id_col: 'USUBJID',
+    time_col: 'VISITN',
+    measure_col: 'TEST',
+    value_col: 'STRESN',
+    unit_col: 'STRESU',
+    normal_col_low: 'STNRLO',
+    normal_col_high: 'STNRHI',
+    start_value: null,
+    groups: [
```

Comment:
I wonder if the default here should just be "none" and the users can configure additional groups as desired